src/utils/validators.js

```diff
@@ -16,6 +16,8 @@ export const URL_REGEX_PREFIX = "^(https://)?(www.)?("
 export const URL_REGEX_SUFFIX = ".com/)([a-zA-Z0-9_-]+([/.])?)+$"
 export const TELEGRAM_REGEX = "telegram|t).me/([a-zA-Z0-9_-]+([/.])?)+$"
 export const TIKTOK_REGEX = ".com/@)([a-zA-Z0-9_-]+([/.])?)+$"
+export const DOMAIN_NAME_REGEX =
+  "(?:[a-z0-9](?:[a-z0-9-]{0,61}[a-z0-9])?\\.)+[a-z0-9][a-z0-9-]{0,61}[a-z0-9]"
```

Comment:
Hmm out of curiosity, why do we need the `{0,61}`

Reply:
[RFC1035](https://www.rfc-editor.org/rfc/rfc1035) specifies that each label is limited to 63 characters, then there is a `[a-z0-9]` before and after the `{0,61}` part, so 63 - 2 = 61.